### DISCUSSION TITLE
a lot of nc_* tables in my project [Jan 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4903] [Discord Threads]
> 🙋‍♂️ **charlietrigger**  
#### 2: are you supposed to use a different DB for the instance vs a project? So should you have db_nocodb and db_project1 and let nocodb save all nc_ tables in db_nocodb? Now I have this in one db, both project and nocodb instance. would this cause issues other than nc_* tables showing up in UI?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1052896205123887145/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### did you create a project by connecting to an external database which is same as NC_DB?  
#  
> 🙋‍♂️ **charlietrigger**  
#### yes, that is correct  
#  
> 🙋‍♂️ **charlietrigger**  
#### is that not the intented way?  
#  
> 🙋‍♂️ **wkw**  
#### NC_DB is holding the meta data. ideally you shouldn't store other data on NC_DB.  
#  
> 🙋‍♂️ **charlietrigger**  
#### ah okay. Thank you for the reply. 😄  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1052896205526548542)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
